pyflakes/checker.py

```diff
@@ -453,6 +453,17 @@ def getParent(self, node):
             if not hasattr(node, 'elts') and not hasattr(node, 'ctx'):
                 return node
 
+    def has_parent_type(self, node, cls):
+        """Check if node has any parent of type cls."""
+        while node:
+            try:
+                node = self.getParent(node)
```

Comment:
I ask because `CONTINUE` below uses `node.parent`. I can't remember if I didn't use `getParent` on purpose or not.